### DISCUSSION TITLE
Running garbo with low adv left today

### DISCUSSION THREAD

**Discussion by asafaq** [loathers/garbage-collector#1981] [Q&A]
If I run garbo with say 40 turns left, will it know how to handle them? Or will it do its usual to acquire various items and buffs and start barfing without quite covering the expenses?
Or will it act differently and get what it can for 40 adv?

**Comment by spaghetti-squash** [comment]
It'll do its usual, but it'll only aim to get 40 turns of buffs (or whatever)
